extensions/gdx-tools/src/com/badlogic/gdx/tools/texturepacker/TexturePacker.java

```diff
@@ -201,7 +201,7 @@ private void writeImages (File outputDir, String scaledPackFileName, Array<Page>
 		File packDir = packFileNoExt.getParentFile();
 		String imageName = packFileNoExt.getName();
 
-		int fileIndex = 1;
+		int fileIndex = 0;

```

Comment:
I like this better, but it is a breaking change. It might be best if this could be configured with a flag, to minimize breakage for older code. If `legacyPages` is true, it uses the old behavior with `fileIndex = 1`; otherwise, it uses 0. This could probably be a static variable without any issue, because the TexturePacker I think only runs on desktop platforms, and if you set it to legacy you probably want to keep it on legacy. If legacy mode is on by default, then this no longer is a breaking change.

Reply:
Should it also exclude the first page from being given an index (like the current behaviour) or only change the starting value?

Reply:
It would avoid being a breaking change if it maintains the same behavior as before the PR, by default. As long as there's an option, though, you could go wild with changes if people opt in.